### PR TITLE
node_options process assignment refactor

### PR DESCRIPTION
**Pull request by alvarorm22**
**Description of the change**

We need to define node options in each container to allow to pass a default --max-old-space-size and set default values if needed

**Benefits**

Allow to pass node_options configurations to each node component

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

### CODE REVIEW

**Comment by alvarorm22** [comment]
https://app.shortcut.com/cartoteam/story/263922/include-node-options-in-selfhosted-components

**Comment by jvillarf** [comment]
https://github.com/CartoDB/cloud-native/pull/9668/files

File: chart/values.yaml [hunk 1]
```diff
@@ -1440,6 +1458,12 @@ mapsApi:
   ##
   containerPorts:
     http: 8002
+  ## @param mapsApi.nodeProcessMaxOldSpacePercentage Memory percentage for node process max-old-space-size param https://nodejs.org/api/cli.html#--max-old-space-sizesize-in-megabytes
+  ##
+  nodeProcessMaxOldSpacePercentage: 75
+  ## @param mapsApi.defaultNodeProcessMaxOldSpace Default memory for node process max-old-space-size param https://nodejs.org/api/cli.html#--max-old-space-sizesize-in-megabytes
+  ##
+  defaultNodeProcessMaxOldSpace: 1536
```
Comment (jvillarf) [inline chart/values.yaml:1466]:
```suggestion
  defaultNodeProcessMaxOldSpace: 1875
```

File: chart/values.yaml [hunk 2]
```diff
@@ -2986,6 +3022,12 @@ workspaceApi:
   ##
   containerPorts:
     http: 8001
+  ## @param workspaceApi.nodeProcessMaxOldSpacePercentage Memory percentage for node process max-old-space-size param https://nodejs.org/api/cli.html#--max-old-space-sizesize-in-megabytes
+  ##
+  nodeProcessMaxOldSpacePercentage: 75
+  ## @param workspaceApi.defaultNodeProcessMaxOldSpace Default memory for node process max-old-space-size param https://nodejs.org/api/cli.html#--max-old-space-sizesize-in-megabytes
+  ##
+  defaultNodeProcessMaxOldSpace: 768
```
Comment (jvillarf) [inline chart/values.yaml:3030]:
```suggestion
  defaultNodeProcessMaxOldSpace: 1020
```

**Review by jvillarf — CHANGES_REQUESTED** [review]
A couple of suggestions to be aligned with https://github.com/CartoDB/cloud-native/wiki/Container-Resources

**Review by jvillarf — APPROVED** [review]

**Pull request merged**